integrations/kube-agent-updater/go.mod

```diff
@@ -5,14 +5,14 @@ go 1.19
 require (
 	github.com/Masterminds/sprig/v3 v3.2.3
 	github.com/docker/distribution v2.8.2+incompatible
-	github.com/google/go-containerregistry v0.14.1-0.20230409045903-ed5c185df419
+	github.com/google/go-containerregistry v0.15.2
 	github.com/gravitational/trace v1.2.1
 	github.com/jonboulle/clockwork v0.4.0
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/sigstore/cosign/v2 v2.0.2
-	github.com/sigstore/sigstore v1.6.4
+	github.com/sigstore/cosign/v2 v2.1.0
```

Comment:
cosign 2.1.0 says: "Breaking Change: The predicate is now a required flag in the attest commands, set via the --type flag." (Whatever that means.)